tests/dist/test_dist_objects.py

```diff
@@ -117,3 +141,10 @@ def test_dist_objects(net_type, num_servers, num_clients, hetero, shared_mem):
     for p in procs:
         p.join()
         assert p.exitcode == 0
+
+def teardown():
```

Comment:
this will be called automatically by pytest after tests are done?

Reply:
Yes that's correct, at the end of all tests (unlike pytest fixtures who are after/before each test)